src/main/resources/com/puppycrawl/tools/checkstyle/filters/messages.properties

```diff
@@ -0,0 +1 @@
+name.invalidPattern=Name ''{0}'' must match pattern ''{1}''.
```

Comment:
what forcing you to add this file and such property?

Reply:
see guidance at https://github.com/checkstyle/checkstyle/pull/14784#issuecomment-2049962607

Reply:
done